dpdispatcher/dpcloudserver/client.py

```diff
@@ -0,0 +1,245 @@
+import json
+import os
+import time
+import urllib.parse
+import humps
```

Comment:
As `humps` is an optional dependency, when it's not installed the error will throw.

Reply:
thanks , I have remove the dependency.